### DISCUSSION TITLE
Should the "Play / Pause" binding work on the results screen?

### DISCUSSION THREAD

**Discussion by Zylviij** [ppy/osu#16972] [Q&A] [answered]
With or without the "Now playing" overlay displayed, I cannot pause audio playback during the results screen, is this intended behavior?

**Comment by Zylviij** [comment]
(Default keybindings tested)

**Comment by bdach** [comment, marked as answer]
similar/related: https://github.com/ppy/osu/discussions/15131

not sure i have a formed opinion on this myself.

**Reply by peppy** [reply]
Should be resolved via https://github.com/ppy/osu/pull/16977.
